snapshot/snapshot.go

```diff
@@ -217,6 +219,10 @@ func NewSnapshotter(ctx context.Context, cfg *config.Config) (snapshots.Snapshot
 }
 
 func (o *snapshotter) Cleanup(ctx context.Context) error {
+	metricBeginTime := time.Now()
+	defer func() {
+		go collector.CollectSnapshotterMetrics(metricBeginTime)
```

Comment:
Please try Prometheus native API to do this https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Timer

Reply:
Done. Thanks.